scripts/shared/clusters.sh

```diff
@@ -125,14 +125,18 @@ function deploy_ovn_cni(){
 }
 
 function deploy_kind_ovn(){
+    local OVN_SRC_IMAGE="quay.io/vthapar/ovn-daemonset-f:latest"
     export K8s_VERSION="${version}"
     export NET_CIDR_IPV4="${cluster_CIDRs[${cluster}]}"
     export SVC_CIDR_IPV4="${service_CIDRs[${cluster}]}"
     export KIND_CLUSTER_NAME="${cluster}"
-    export OVN_IMAGE="quay.io/vthapar/ovn-daemonset-f:latest"
     export REGISTRY_IP="${registry_ip}"
 
-    docker pull "$OVN_IMAGE"
+    export OVN_IMAGE="localhost:5000/ovn-daemonset-f:latest"
```

Comment:
Is `OVN_IMAGE` expected to be used anywhere else? We might not need to `export` it.

Reply:
It is used by `kind.sh` It doesn't have any param for it and expects this to be exported.